n_request.c

```diff
@@ -54,10 +54,10 @@ static J *errDoc(const char *errmsg)
         JAddStringToObject(rspdoc, "src", "note-c");
 
         if (suppressShowTransactions == 0) {
-            // Since we're already allocating...
-            char * err = JConvertToJSONString(rspdoc);
-            NOTE_C_LOG_ERROR(err);
-            _Free(err);
+            _DebugWithLevel(NOTE_C_LOG_LEVEL_ERROR, "[ERROR] ");
+            _DebugWithLevel(NOTE_C_LOG_LEVEL_ERROR, "{\"err\":\"");
+            _DebugWithLevel(NOTE_C_LOG_LEVEL_ERROR, errmsg);
+            _DebugWithLevelLn(NOTE_C_LOG_LEVEL_ERROR, "\",\"src\":\"note-c\"}");
```

Comment:
This is a reversion to what it used to be, but more appropriately wrapped by our logging semantics.